include/ignition/gazebo/EntityComponentManager.hh

```diff
@@ -696,6 +696,12 @@ namespace ignition
           const Entity _entity, const ComponentTypeId _type,
           gazebo::ComponentState _c = ComponentState::OneTimeChange);
 
+      /// \brief All future entities will have an id that starts at _offset.
+      /// This can be used to avoid entity id collisions, such as during log
+      /// playback.
+      /// \param[in] _offset Offset value.
+      public: void SetEntityCreateOffset(uint64_t _offset);
```

Comment:
Mind adding a unit test for this function? It could help covering the lines on `CreateEntity()` which are currently uncovered.

Reply:
Added test: 8a2b44cf1122c67287ac079a20efac4cb8ffd422